### PR TITLE
Linaria 6

### PR DESCRIPTION
**Pull request by amanmahajan7**
https://github.com/callstack/linaria/releases/tag/linaria%406.0.0
https://wyw-in-js.dev/

### CODE REVIEW

File: src/Cell.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { memo } from 'react';
-import { css } from '@linaria/core';
+import { css } from '@wyw-in-js/core';
```
Comment (nstepien) [inline src/Cell.tsx:2]: Can't we keep these imports as `@linaria/core`? The package still exists

Comment (amanmahajan7) [reply src/Cell.tsx:2]: yeah, was just fixing it

**Comment by amanmahajan7** [comment]
`styles.css` is the same but `bundle.js` is completely different. Were we packaging it incorrectly before?

image

**Comment by nstepien** [comment]
There shouldn't be any remaining css`` in the output 🤔

**Comment by amanmahajan7** [comment]
May be I incorrectly generated  the bundle in the main branch
image

**Comment by amanmahajan7** [comment]
Anyway looks like the output is the same

File: package.json [hunk 2]
```diff
@@ -64,8 +64,7 @@
     "@biomejs/biome": "1.4.1",
     "@faker-js/faker": "^8.0.0",
     "@ianvs/prettier-plugin-sort-imports": "^4.0.2",
-    "@linaria/rollup": "^5.0.3",
-    "@linaria/vite": "^5.0.3",
+    "@linaria/core": "^6.0.0",
```
Comment (amanmahajan7) [inline package.json:67]: This needs to be installed now

**Review by nstepien — APPROVED** [review]

**Pull request merged**